paimon-core/src/test/java/org/apache/paimon/table/AppendOnlyFileStoreTableTest.java

```diff
@@ -117,6 +118,61 @@ public void testBatchFilter() throws Exception {
                                 "2|21|201|binary|varbinary|mapKey:mapVal|multiset"));
     }
 
+    @Test
+    public void testBatchFilterWithExecution() throws Exception {
```

Comment:
Is it necessary to add a same test in `PrimaryKeyFileStoreTableTest`?

Reply:
I think it is the same, so I think it is OK to be just here.

Reply:
Alright.